roles/openshift_hosted/tasks/storage/glusterfs.yml

```diff
@@ -79,14 +79,7 @@
       - REGISTRY_STORAGE_MAINTENANCE_READONLY_ENABLED: 'true'
   when: openshift.hosted.registry.storage.glusterfs.swap
 
-- name: Unmount registry volume
+- name: Unmount registry volume and clean up mount point/fstab
   mount:
-    state: unmounted
-    name: "{{ mktemp.stdout }}"
-
-- name: Delete temp mount directory
```

Comment:
If we no longer need this temp directory, let's clean it up.

Reply:
`absent` in the mount module does this for us

Reply:
Looks like you're right.  But it also looks like using 'state: absent' will not actually unmount the mount according to the docs:

'absent and present only deal with fstab but will not affect current mounting.'

Looks like we need two statements here.

Reply:
yeah that's what I thought too. and then we found a trail of "wrong". It starts [here](https://github.com/openshift/openshift-ansible/issues/6004#issuecomment-342542201).